doc/nrf/device_guides/working_with_nrf/nrf70/developing/fw_patches_ext_flash.rst

```diff
@@ -0,0 +1,133 @@
+.. _ug_nrf70_developing_fw_patch_ext_flash:
+
+Firmware patches in the external flash
+######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+Overview
+********
+
+The nRF70 Series firmware patch is memory intensive and is by default part of the internal flash.
+In order to save flash memory for resource constrained hosts, the nRF70 Series offers a feature to move firmware patches to the external flash, if the external flash is available.
+
+Pre-requisites
+--------------
+
+The following conditions must be met to use this feature:
+
+- The external flash must be available and configured in the device tree.
+- The external flash must be large enough to store the firmware patches, maximum size is 128 KB.
+
+Constraints
+-----------
+
+The following constraints apply to this feature:
+
+- Access to the external flash is only supported via the QSPI interface for now.
+- In nRF SoC's XIP feature is only available via the QSPI interface only.
+
+
+Supported platforms
+-------------------
+
+The following platforms are supported:
+
+- nRF5340 SoC or DK
+- nRF52840 SoC or DK
+
+.. note:: Due to constraints mentioned above, the nRF7002 SoC or DK is not supported as it needs access to external flash via the SPI interface.
+
+Details
+*******
+
+There are two ways to move the firmware patch to the external flash:
+
+* XIP Supported external flash
+* non-XIP Supported external flash
+
+
+XIP Supported external flash
+----------------------------
+If the external flash supports XIP, then the firmware patch can be relocated to the external flash and executed from there.
```

Comment:
You can choose to not use XIP even if the external flash supports XIP. I suggest that we use "application" instead
```suggestion
If the application supports XIP from external flash, then the firmware patch can be relocated to the external flash and executed from there.
```